src/data/ab-attrs/move-flag-power-boost-ab-attr.ts

```diff
@@ -0,0 +1,25 @@
+import type { PokemonAttackCondition } from "#app/@types/PokemonAttackCondition";
+import type { MoveFlags } from "#enums/move-flags";
+import { MovePowerBoostAbAttr } from "./move-power-boost-ab-attr";
+
+/**
+ * Ability attribute that boosts the power of a move by a factor if it has a specified flag
+ * +---------------+---------------+------------+
```

Comment:
```suggestion
 * ```
 * +---------------+---------------+------------+
```
If you're going to put these tables in tsdocs they need to have the backticks above and below them, otherwise they look like this:
![image](https://github.com/user-attachments/assets/8c4af0b4-18de-4617-8515-23c7e3b53f2f)